src/Common/MultiVersion.h

```diff
@@ -36,6 +36,16 @@ class MultiVersion
     {
     }
 
+    /// There is no copy constructor because only one MultiVersion should own the same object.
```

Comment:
MultiVersion by its design should not allow copying. It's even constructed from `unique_ptr`.